sdk/search/Azure.Search.Documents/src/Serialization/JsonSerialization.cs

```diff
@@ -72,9 +70,8 @@ public static JsonSerializerOptions AddSearchConverters(this JsonSerializerOptio
             options.Converters.Add(SearchDateTimeOffsetConverter.Shared);
             options.Converters.Add(SearchDateTimeConverter.Shared);
             options.Converters.Add(SearchDocumentConverter.Shared);
-#if EXPERIMENTAL_SPATIAL
             options.Converters.Add(new GeoJsonConverter());
```

Comment:
Nit: if `GeoJsonConverter` isn't stateful (they aren't supposed to be, in general), perhaps we should keep a shared instance of this ourselves.